package.json

```diff
@@ -261,27 +261,27 @@
     "coveralls": "2.11.15",
     "css-loader": "0.26.1",
     "enzyme": "2.7.1",
-    "eslint": "3.14.1",
-    "eslint-config-airbnb": "14.0.0",
+    "eslint": "^3.13.0",
```

Comment:
No that's an error. 

I used the script on [this page](https://www.npmjs.com/package/eslint-config-airbnb) to ensure eslint-config-airbnb and friends played nicely. Didn't see the carets. Will fix shortly...

 Thanks for reviewing 😄